courses/dl2/pascal.ipynb

```diff
@@ -442,11 +444,41 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 19,
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bb_voc = [155, 96, 196, 174]\n",
+    "bb_fastai = hw_bb(bb_voc)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
    "metadata": {},
    "outputs": [],
    "source": [
-    "def bb_hw(a): return np.array([a[1],a[0],a[3]-a[1],a[2]-a[0]])"
+    "def bb_hw(a): return np.array([a[1],a[0],a[3]-a[1]+1,a[2]-a[0]+1])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'expected: [155, 96, 196, 174], actual: [155  96 196 174]'"
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "'expected: {}, actual: {}'.format(bb_voc, bb_hw(bb_fastai))"
```

Comment:
Thanks! This would be better done with a python 3.6 f-string for consistency with the rest of the lessons.